DirectEmployers/MyJobs
# PD-3688 Fixing columns in Apply Clicks

## What's this PR do?
Apply Clicks dimension of new site-traffic needs to include Job data along with Goal Data. Fields corresponding to Job Data have been added
## Screenshots (if appropriate)

## Where should the reviewer start?
a new dimension for apply clicks has been added to schema_dimensions.py
## How should this be manually tested?
Check the Apply Clicks dimension of new site traffic report to find Job data(Title, Req ID, Location)
## Any background context you want to provide?
